src/services/refactor.service.js

```diff
@@ -0,0 +1,9 @@
+export const refactorPhoneNumber = (phone = '') => {
```

Comment:
It is not a service, it should be in `utils` folder and be called as the name of the function

Reply:
moved and fixed that